pep-0695.rst

```diff
@@ -2,7 +2,7 @@ PEP: 695
 Title: Type Parameter Syntax
 Author: Eric Traut <erictr at microsoft.com>
 Sponsor: Guido van Rossum <guido@python.org>
-Discussions-To: typing-sig@python.org
+Discussions-To: https://mail.python.org/archives/list/typing-sig@python.org/thread/BB2BGYJY2YG5IWESKGTAPUQL3N27ZKVW/
```

Comment:
Could you also add, at the end (I can't reach it to make an actual suggestion):

```email
Post-History: `20-Jun-2022 <https://mail.python.org/archives/list/typing-sig@python.org/thread/BB2BGYJY2YG5IWESKGTAPUQL3N27ZKVW/>`__
```

Reply:
Done, let's hope I added it in the right place.